x-pack/plugins/ml/public/application/explorer/explorer.js

```diff
@@ -145,6 +146,22 @@ export class Explorer extends React.Component {
   state = { filterIconTriggeredQuery: undefined, language: DEFAULT_QUERY_LANG };
   htmlIdGen = htmlIdGenerator();
 
+  componentDidMount() {
+    const { invalidTimeRangeError } = this.props;
+    if (invalidTimeRangeError) {
+      const toastNotifications = getToastNotifications();
+      toastNotifications.addWarning(
+        i18n.translate('xpack.ml.explorer.invalidTimeRangeInUrlCallout', {
+          defaultMessage:
+            'The time filter changed to the full range for this job due to invalid default time filter. Please check the advanced settings for {field}.',
+          values: {
+            field: ANOMALY_DETECTION_DEFAULT_TIME_RANGE,
```

Comment:
The message either needs changing to remove `for this job` or else you need to pass in the number of jobs selected in the view and change to `for these jobs` if more than one is selected.

Reply:
> I removed the "for this job" part...

Should it be removed from the UI text on the Advanced Settings page too?